### PR TITLE
Adds blacklist to the Pneumatic Cannon because im unfun

### PR DESCRIPTION
**Pull request by JamieD1**
Adds blacklist to the Pneumatic Cannon because i'm unfun

:cl:  
rscadd: Adds blacklist to the Pneumatic Cannon because i'm unfun
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
has someone actually put a super matter sword into the cannon before in a normal round or are you just putting it in so batons arent lonely in the list

**Comment by LazennG** [comment]
baton confirmed to be on the same level of danger as sm sword

File: code/game/objects/items/pneumaticCannon.dm [hunk 1]
```diff
@@ -102,6 +106,10 @@
 /obj/item/pneumatic_cannon/proc/can_load_item(obj/item/I, mob/user)
 	if(!istype(I))			//Players can't load non items, this allows for admin varedit inserts.
 		return TRUE
+	if(blacklist_items && is_type_in_list(I, blacklist_items))
+		if(user)
+			to_chat(user, span_warning("You are prevented from putting [I] into [src]!"))
```
Comment (ToasterBiome) [inline code/game/objects/items/pneumaticCannon.dm:111]:
```suggestion
			to_chat(user, span_warning("No."))
```

**Comment by maxion12345** [comment]
cope sounds like an ided to me

**Pull request merged**